composer.json

```diff
@@ -21,14 +22,7 @@
     },
     "autoload": {
         "psr-4": {
-            "Google\\Api\\": "src/Api",
             "Google\\ApiCore\\": "src/ApiCore",
```

Comment:
If you're open to it, we should be able to put everything in the `ApiCore` directory right into `src` now and replace this line with `"Google\\ApiCore\\": "src"`

Reply:
Was just thinking the same thing actually :-)
Done. PTAL at src/Version, which needed a change.

Reply:
Great catch 👍

Reply:
Should I do the same in tests?

Reply:
Yes, SGTM :)